ui/revenuecatui/src/main/kotlin/com/revenuecat/purchases/ui/revenuecatui/InternalPaywallView.kt

```diff
@@ -5,7 +5,7 @@ import androidx.compose.runtime.Composable
 import androidx.compose.runtime.collectAsState
 import androidx.lifecycle.viewmodel.compose.viewModel
 import com.revenuecat.purchases.Offering
-import com.revenuecat.purchases.ui.revenuecatui.templates.template2.Template2
+import com.revenuecat.purchases.ui.revenuecatui.templates.Template2
```

Comment:
Just moving `Template2` out to the `templates` package. If/when we extract views out of the `Template2` file, we can reconsider creating packages for each template